src/main/java/build/buildfarm/instance/stub/StubInstance.java

```diff
@@ -240,23 +251,90 @@ public void write(byte[] b, int off, int len) throws IOException {
 
   @Override
   public InputStream newStreamInput(String name) {
-    throw new UnsupportedOperationException();
+    Iterator<ReadResponse> replies = bsBlockingStub
+      .get()
```

Comment:
use four spaces indentation here?

Reply:
I need to put a linter in place for this.  Any recommendations?